src/App.js

```diff
@@ -1,15 +1,38 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList'
+// import Todo from './components/TodoComponents/Todo'
+import TodoForm from './components/TodoComponents/TodoForm'
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state = {
+      listArray: [],
+    };
+  }
+
+  addListItem = newItem => {
+    this.setState({ listArray: [...this.state.listArray, newItem] })
+  }
+
+  clearAllItems() {
+    this.setState({
+      emptyArray: []
+    })
```

Comment:
This method will actually just assign a new (empty) array to your state.

If you want to clear the actual todo array, simply set the actual value of the state property `listArray` to an empty array, like so: `this.setState({ listArray: [] });`